packages/mainsail/source/confirmed-transaction.dto.ts

```diff
@@ -192,6 +192,10 @@ export class ConfirmedTransactionData extends DTO.AbstractConfirmedTransactionDa
 		}));
 	}
 
+	public override methodHash(): string {
+		return this.data.data.slice(0, 10);
```

Comment:
best to add a comment in the code why this needs 10 while the other is 8, as that isn't clear from the code at a glance

Reply:
Added comments